src/main/java/org/mifos/connector/channel/camel/utils/AMSUtils.java

```diff
@@ -0,0 +1,50 @@
+package org.mifos.connector.channel.camel.utils;
+
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+import java.util.List;
+
+
+@Component
+public class AMSUtils {
+
+    private Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    @Autowired
+    private AMSProps amsProps;
+
+    List<AMSProps.AMS> ams;
+   String accountid;
+   String foundationalid;
+   String amsdefault;
+   String amssecondary;
+
+    public AMSUtils(){
+    }
+
+    @PostConstruct
+    public void postConstruct(){
+       ams  =  amsProps.getGroups();
+       ACCOUNTID = ams.get(0).getIdentifier();
+       FOUNDATIONALID = ams.get(1).getIdentifier();
+        AMS_DEFAULT =ams.get(1).getValue();
+        AMS_SECONDARY = ams.get(0).getValue();
+    }
+
+    public static final String MSISDN = "MSISDN";
+    public  String ACCOUNTID;
+    public  String  FOUNDATIONALID;
+    public  String AMS_SECONDARY;
```

Comment:
We discussed that secondary/primary doesn't make sense.

Reply:
Yes, I missed this, will rename it.